fuel-client/src/client/schema/tx.rs

```diff
@@ -280,13 +282,10 @@ pub mod tests {
     use crate::client::schema::Bytes;
     use fuel_vm::fuel_types::bytes::SerializableVec;
 
-    pub mod transparent_receipt;
-    pub mod transparent_tx;
-
     #[test]
     fn transparent_transaction_by_id_query_gql_output() {
         use cynic::QueryBuilder;
-        let operation = transparent_tx::TransactionQuery::build(TxIdArgs {
+        let operation = TransactionQuery::build(TxIdArgs {
```

Comment:
Okay, I found! It is because you changed the type here=) And question why?

Reply:
Reverted! :)